backend/setup/index.js

```diff
@@ -39,9 +39,7 @@ module.exports = function (context, req) {
   const body = req.body;
   body.id = uniqid();
 
-  if (!body.pagetitle || body.pagetitle === "") {

```

Comment:
Why are you no longer checking for a valid page title?

Reply:
Is the pagetitle no longer a part of the backend request?

Reply:
Yup— since the page title is now just dependent on the document title & gets set automatically, there's no need for the user to submit a page title for the initial setup.